web_console/about-administrator-perspective.adoc

```diff
@@ -0,0 +1,30 @@
+:_content-type: ASSEMBLY
+[id="about-administrator-perspective"]
+= About the Administrator perspective in the web console
+include::_attributes/common-attributes.adoc[]
+:context: about-administrator-perspective
+
+toc::[]
+
+The {product-title} web console provides two perspectives; the *Administrator* perspective and the *Developer* perspective.
```

Comment:
You have already included a variation of this sentence with the web console overview assembly file, so I don't think you need to restate it.